.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,21 @@
 
 ## Checklist
 
-- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
-- [ ] The title of the PR is clear and informative
-- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
+[ ] I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and below information
+
+- **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to `generation` branch. 
+- **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
+- **SHOULD** update `ChangeLog.md` file(s) appropriately
     - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
-    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
-- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
-- [ ] If applicable, the changes made in the PR have proper test coverage
-- [ ] For public API changes to cmdlets:
-    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
-        - {Please put the link here}
-    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
+    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into `generation` branch.
+    - Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
+- **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_) with following situations
```

Comment:
Redirect the developer to how to connect their Microsoft account to GitHub account 
```suggestion
- **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
```

Reply:
Updated.